### ISSUE TITLE
I can haz Slack invite?

### ISSUE THREAD

**Issue by jhnklly** [maptime/maptime-admin#109]
Add @jhnklly to Slack please;thx.

**Comment by bfreeds** [comment]
also @bfreeds please!

**Comment by jhnklly** [comment]
assuming you need email to add to Slack: jhnklly@gmail.com

**Comment by geobrando** [comment]
Hey there.  Are either of you currently or would like to be organizers of a new chapter?  If not, you could join the Gitter chat room and interact with other maptimers there.  There's not a lot of folks in there now but hopefully that will change.

[![Join the chat at https://gitter.im/maptime/maptime-admin](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/maptime/maptime-admin?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

**Comment by bfreeds** [comment]
Hey Brandon, thanks for the link to the chat.  I'm with the NYC chapter---frequent attendee, starting to help out with organizing.  

Does most of the conversation happen on Gitter or on Slack?

**Comment by jhnklly** [comment]
Thanks Brandon. I'm in a similar position to Brandyn's (but in SF)--regular attendee, and leading the presentation at the next maptime here. I'm a little confused as to the communication channels. I asked for access to Slack because people at the Maptime Summit sometimes answered questions by saying "ask on Slack." Then when someone said "How do I get on Slack?" the reply was "create an issue on github."

Is the gitter channel also only for admins? (As the channel name "maptime-admin" suggests.) Should we start a channel open to all maptime attendees, for mapping questions that are not administration related? I'll follow up this comment with another of actual questions to discuss on a community (non-admin) channel.

Great summit Monday--thanks to everyone involved!

**Comment by jhnklly** [comment]
Meta question: where's the best place to discuss this:

Is there a place for maptime folks to get help from other maptime folks? Options (not mutually exclusive) :
    1. Slack
    2. Gitter
    3. Twitter
    4. stackoverflow.com / gis.stackexchange.com

For 1 and 2, it seems we'd want to create a separate, not-admin-exclusive channel, yeah?

I believe (3) was suggested by Sarah Levine, in the form of a twitter hashtag like #NoMapQuestionsAreStupid. I think that's a great idea. (I'd like to leave the exact wording of that # to Sarah.)

(4) In discussions with new coders/mappers at the summit I recommended [gis.stackexchange.com](http://gis.stackexchange.com/) & [stackoverflow](http://www.stackoverflow.com/) as excellent resources when stuck on technical problems. Some people suggested that we start a "maptime" tag there as a way to find/track similar Q&A. A few benefits/issues:
   a. Reaches a wide audience: we'd get answers from folks that aren't aware of maptime, and would help those who don't know about maptime find us.
    b. Great place to preserve Q/A--stackexchange is usually at the top of search engine results.
    c. Easier than maintaining a separate maptime FAQ. (I think http://maptime.io/lessons-resources/ serves a somewhat different purpose, and should continue as such.)
    d. Issue: since stackexchange is a broader, pre-existing community, we'll have to be aware of their established culture and [scope](http://gis.stackexchange.com/help/on-topic). (Once in a while stackoverflow gets a bit unwelcoming with the "out of scope" flagging.)

I'd love to hear what others think about all of the above. Can a maptime admin suggest the best method for that?

**Comment by geobrando** [comment]
I remember that. We have a similar situation in DC (regular presenter, not
an "organizer"). Gitter is a solution but not sure it will gain traction.
It is not exclusive for repo-associated rooms but is for
organization-associated rooms. That is why we have one for maptime-admin.
Potentially, each chapter could maintain their own gitter room. That is my
wish.

Another possibility is "guest" accounts that will come when we upgrade our
Slack plan. Not sure if guest accounts count as members to Slack.

See #102

**Comment by lyzidiamond** [comment]
Thanks for starting this conversation, @jhnklly and @bfreeds. I think all of this depends upon the purpose of the communication.

1. Slack is an organizing channel for admins, and the conversations there are generally limited to logistical issues around individual chapter meetups and volunteering for admin. If you're interested in volunteering for admin, or you have become an organizer for your chapter, Slack is for you.
2. For technical questions, I would suggest Twitter, StackOverflow, or the #maptime channel in IRC (on freenode). Gitter seems to be an interesting tool, but it seems to mimic a lot of behavior that IRC already has. @almccon, what was the impetus to use Gitter vs. IRC?
3. I'd love to see more general chat happen between folks in the Maptime community. I personally watch the #maptime hashtag on Twitter and we try to facilitate discussions by retweeting maptime-related activity. If you're interested in facilitating communication within your own chapter, there are avenues for that: your GitHub repo, an IRC chat room, or a mailing list. For general map-related questions, Twitter is great, and I tend to tag my tweets with questions with #maptime.

I'm not sure if this is helpful, and I'm definitely open to thinking about how we can open up communications more. IRC seems like the best bet to me, to be honest.

**Comment by bfreeds** [comment]
I appreciate your thoughts and tips @lyzidiamond @geobrando and @jhnklly on the discussion avenues.  If slack is mostly used for admins, then I probably don't need to be on it for now (although as I get more involved with my local chapter and learn the ropes I would love to begin helping out with Admin).  I hadn't realized that Slack had a user-based fee-fingers crossed for free accounts!

Honestly I'm still getting the hang of twitter, but in my mind I was thinking of it as more social/stream of consciousnesses and event-based discussions.  I'm not sure if it is as good for collaborating on specific and long-term projects.

Basically, I just hope that there is a good avenue that everyone is on board with to continue the type of discussions during the unconference sessions.  Perhaps Gitter would be good within a repo Maptime Organizers use to develop tutorials (like the one requested by Dan #110).

**Comment by geobrando** [comment]
@bfreeds 
> Basically, I just hope that there is a good avenue that everyone is on board with to continue the type of discussions during the unconference sessions.

Agreed.  This will likely be gitter or irc (I'd completely forgetten that we had one).  A #maptime hashtag on twitter is good for short interactions but not sufficient for discussions. As Lyzi, suggested, local chapters could choose to use github issues for discussion. I haven't had much luck getting people to post there in DC.

> I hadn't realized that Slack had a user-based fee-fingers crossed for free accounts!

We're on a free account but hoping to take advantage of features that come with upgrade to standard plan, in which you have to pay per user/per month if you don't qualify for the discount afforded to non-profits.

@lyzidiamond In my opinion, gitter (which is almost identical to Slack) is much more feature-rich and user-friendly.  Freenode has a web interface, which levels the playing field a bit.  The only downside of gitter is that it requires a github account.  But we're trying to encourage github adoption so that may not be a big deal.

**Comment by bfreeds** [comment]
@geobrando  This is really insightful, thank you.  I agree that Github Issues and Gitter could be a great solution (at least while we wait on the Slack decision).  I'm fine with closing this, if others agree.  Would like to hear from other Maptime chapters how they would like to communicate.

**Comment by almccon** [comment]
Following up on @lyzidiamond's question from 5 days ago: we didn't really "decide" to use gitter over IRC. We have both, and nobody uses either one. :)

Gitter is new and untested, but so far it seems a bit easier to get started with, rather than IRC. And it has the benefit of requiring people to sign up for github, which we want to encourage, as @geobrando points out.

I should also add that for local chapters who want to have lots of loosely-affiliated semi-organizers, they can create their own slack channel. MaptimeLA did this. It doesn't quite solve the problem for people like @bfreeds and @jhnklly. Although maybe MaptimeNYC might want their own Slack channel, who knows.

**Comment by cityhubla** [comment]
Hi maptimeers, thanks for having this on git, I was reading some docs on the org structure/bylaws and found this admin section to the group. I want to thank you all for making this happen, and we at maptimeLA wanted to ensure we are in keeping with the mission of Maptime. 

We did create a slack channel for semi-non-organizers to participate on projects/tutorials, more so after the HackforLA hackathon experience. I have to say that our success was because of the open learning experience we enjoy during through our maptime meetups. 

Our last meetup was a recap of what we did, how we did it and reflections on improving our learning through these activities to which one of our organizers made a guide on **"How to work with a lot of people of all levels on awesome projects", [Trail Guide](https://github.com/maptimeLA/trail-guide)**, the guide was updated with the conversations we had with everyone there, and made the hackathon a learning experience. The slack channel would hopefully continue participation offMaptime with some of the projects (which we're figuring out how to document the steps and make tutorials of) we are working on and encourage folks to take the lead and also learn the tools for communication (slack, glitter, github, smoke signals, carrier pigeons) that gets people to make awesome maps. Any suggestions and ideas would be "pushed" to the org channel so other chapters could use in their group efforts.

That's the happenings going down in SoCal!

**Comment by jhnklly** [comment]
Thanks for all the input, everyone. Seems like there's a consensus forming toward Gitter--sounds good to me. (Getting started with IRC is probably as much a hurdle as getting on github, and github is a fairly essential tool.) I'll leave this thread open another day or two (or longer if comments continue), then look for discussions on Gitter. (And gh-issues/stackexchange/twitter... :)

**Comment by geobrando** [comment]
@cityhubla Very interesting. Thanks for that.

@jhnklly Yes, I would say we should encourage chapters to either create a Gitter room for their chapter repo or a separate Slack account if they're looking for a platform for communication between organizers and contributors/participants.

In terms of communication between maptimers worldwide, we'll continue to have both the main Gitter room I linked to above and IRC and see how that shakes out.  

@almccon & @lyzidiamond : If you agree that the maptime-admin Gitter room is preferred, maybe we can ask organizers to make a push for their participants to sign up for a github account and join the room as well as mention the above options for local communication.

I'll close this and we can talk more about that on Slack.
